inst/templates/gitbook.html

```diff
@@ -123,12 +123,22 @@ <h2 class="subtitle"><em>$subtitle$</em></h2>
 $endif$
 $for(author)$
 $if(author.name)$
+<style type="text/css">
+  #header p.author {
```

Comment:
The id `header` could have a prefix (see line 119 above), so I'd like to just define `p.author` without the parent id:

```suggestion
  p.author {
```